blueprints/soci-snapshotter/README.md

```diff
@@ -150,37 +182,81 @@ spec:
     --//
 ```
 
+### Bottlerocket
+
+SOCI is integrated into Bottlerocket latest AMIs and simplify setup and configuration through Bottlerocket APIs as in the example below.
+
+In Bottlerocket, SOCI's data dir is configured at `/var/lib/soci-snapshotter`, to take advantage of instances with NVMe disks, we will need to configure ephemeral storage through Bottlerocket's Settings API, replacing `instanceStorePolicy: RAID0` with `[settings.bootstrap-commands.k8s-ephemeral-storage]` as you can see below, we added `/var/lib/soci-snapshotter` as a bind dir.
+
+```yaml
+apiVersion: karpenter.k8s.aws/v1
+kind: EC2NodeClass
+metadata:
+  name: soci-snapshotter-br
+...
+...
+spec:
+...
+...
+  userData: |
+    [settings.container-runtime]
+    snapshotter = "soci"
+    [settings.container-runtime-plugins.soci-snapshotter]
+    pull-mode = "parallel-pull-unpack"
+    [settings.container-runtime-plugins.soci-snapshotter.parallel-pull-unpack]
+    max-concurrent-downloads-per-image = 10
+    concurrent-download-chunk-size = "16mb"
+    max-concurrent-unpacks-per-image = 10
+    discard-unpacked-layers = true
+    [settings.bootstrap-commands.k8s-ephemeral-storage]
+    commands = [
+        ["apiclient", "ephemeral-storage", "init"],
+        ["apiclient", "ephemeral-storage" ,"bind", "--dirs", "/var/lib/containerd", "/var/lib/kubelet", "/var/log/pods", "/var/lib/soci-snapshotter"]
+    ]
+    essential = true
+    mode = "always"
+```
+
 ## Results
 
 Wait until the pods from the sample workload are in running status:
 ```sh
 > kubectl wait --for=condition=Ready pods --all --namespace default --timeout=300s
-pod/vllm-85b4fbdd49-k88gg condition met
-pod/vllm-soci-744dfdbcfb-d4bwj condition met
+pod/vllm-59bfb6f86c-9nfxb condition met
+pod/vllm-soci-6d9bfd996d-vhr4j condition met
+pod/vllm-soci-br-74b59cc4bd-rq8cw condition met
 ```
 
 The sample workload deploys two Deployments running [Amazon Deep Learning Containers for vLLM](https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/dlc-vllm-x86-ec2.html), one using SOCI parallel pull/unpack mode and one remains using the default containerd implementation.
```

Comment:
I see three deployments now

Reply:
also, wouldn't be useful to see the benefit by letting people know how big the container image is?